content/tracing/setup/go.md

```diff
@@ -80,53 +100,60 @@ ___
 | SQLx                | https://github.com/jmoiron/sqlx                                       | https://godoc.org/github.com/DataDog/dd-trace-go/contrib/jmoiron/sqlx                      |
 
-### OpenTracing API
+___
+
+Make sure to take a look at our integrations package [godoc page][contrib godoc] for an in-depth look.
 
-Datadog APM client that implements an [OpenTracing][3] Tracer.
+## OpenTracing Support
 
-**Initialization**
+Import the [`opentracer` package][opentracing godoc] to expose the Datadog tracer as an [OpenTracing][3] compatible tracer.
 
-To start using the Datadog Tracer with the OpenTracing API, you should first initialize the tracer with a proper `Configuration` object:
+### Example
+
+A basic usage would be:
 
 ```go
+package main
+
 import (
-  // ddtrace namespace is suggested
-  ddtrace "github.com/DataDog/dd-trace-go/opentracing"
-  opentracing "github.com/opentracing/opentracing-go"
+    "github.com/opentracing/opentracing-go"
+
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/opentracer"
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
 func main() {
-  // create a Tracer configuration
-  config := ddtrace.NewConfiguration()
-  config.ServiceName = "api-intake"
-  config.AgentHostname = "ddagent.consul.local"
-
-  // initialize a Tracer and ensure a graceful shutdown
-  // using the `closer.Close()`
-  tracer, closer, err := ddtrace.NewTracer(config)
-  if err != nil {
-    // handle the configuration error
-  }
-  defer closer.Close()
-
-  // set the Datadog tracer as a GlobalTracer
-  opentracing.SetGlobalTracer(tracer)
-  startWebServer()
-}
-```
+    // Start the regular tracer and return it as an opentracing.Tracer. You may use
+    // the same set of options as you normally would with the Datadog tracer.
+    t := opentracer.Start(tracer.WithServiceName("my-service"))
 
-Function `NewTracer(config)` returns an `io.Closer` instance that can be used to gracefully shutdown the `tracer`. It's recommended to always call the `closer.Close(), otherwise internal buffers are not flushed and you may lose some traces.
+    // Stop it using the regular Stop call.
+    defer tracer.Stop()
+    
+    // Set the global OpenTracing tracer.
+    opentracing.SetGlobalTracer(t)
 
-**Usage**
+    // Use the OpenTracing API as usual.
+}
+```
 
-See [Opentracing documentation][4] for some usage patterns. Legacy documentation is available in [GoDoc format][5].
+**Note**: Using the [OpenTracing API][4] in parallel with the regular API or our integrations is fully supported. Under the hood, all of them
+make use of the same tracer. Make sure to check out the [API documentation][opentracing godoc] for more examples and details.
 
 ## Further Reading
 
+Find more information in the [package documentation][tracer godoc] about distributed tracing and sampling priority.
```

Comment:
I don't think we need this.  Instead we should just add a small section for Distributed Tracing like we have on https://docs.datadoghq.com/tracing/setup/java/#sampling-distributed-tracing and https://docs.datadoghq.com/tracing/setup/ruby/#distributed-tracing

cc @gbbr @palazzem

Reply:
I can see in the case of Java it's written right there, but in the case of Ruby it's pretty much the same as here, except it's contained under its own section. I'm fine to go with any option.

Reply:
Done. I've used the Java example to illustrate distributed sampling and used the Ruby example to link to the distributed tracing doc (like now). A mixture of both basically. The Java page is lacking information on distributed tracing.